lib/ansible/modules/cloud/azure/azure_rm_aks.py

```diff
@@ -96,6 +96,92 @@
         type: bool
         default: no
         version_added: 2.8
+    network_profile:
+        description:
+            - Profile of network configuration.
+        suboptions:
+            network_plugin:
+                description:
+                    - Network plugin used for building Kubernetes network.
+                    - This property cannot been changed.
+                choices:
+                    - azure
+                    - kubenet
+            network_policy:
+                description: Network policy used for building Kubernetes network.
```

Comment:
so what should that policy be exactly?

Reply:
which can be used to set  the network rule of pod: https://kubernetes.io/docs/concepts/services-networking/network-policies/

Reply:
ok, so the most likely scenario will be that user will use lookup to load it from file or somewhere else, right?

Reply:
add doc link for user's reference?